src/components/radio/README.md

```diff
@@ -54,6 +54,7 @@ The `md-radio-group` component has no button initially selected.
 | `value` | `any` | The value of this radio button. |
 | `checked` | `boolean` | Whether the radio is checked. |
 | `disabled` | `boolean` | Whether the radio is disabled. |
+| `actionName` | `string` | Used to set the `aria-label` attribute of the underlying input element. |
```

Comment:
I think it is not a good idea to have a *pseudo* `aria-label` attribute on the MdRadio component and at the same time a real one on the underlying input element. It might confuse screen readers and its users. Do you know what I mean? That's why I chose a different name as seen on the [Angular 2 documentation for template syntax](https://angular.io/docs/ts/latest/guide/template-syntax.html) (see Property Binding / We must use attribute binding when there is no element property to bind).

But while taking a look a the current implementation for MdSlideToggle, I see that the property used there is named `ariaLabel`, which is different from `aria-label`. So this subtle difference is probably enough avoid potential issues with screen readers.

So if everyone agrees and doesn't see any more issues, I can change the property name (and add `aria-labelledby` as well).